Detections/SecurityEvent/PotentialBuildProcessCompromise.yaml

```diff
@@ -54,10 +58,49 @@ query: |
   | where BuildProcessTime <= FileEditTime
   | summarize make_set(FileEdited), make_set(FileEditProcess), make_set(FileEditAccount) by timekey, Computer, BuildParentProcess, BuildProcess
   | extend HostCustomEntity=Computer, timestamp=timekey
+  ),
+  (WindowsEvent
```

Comment:
While not a high priority, it would make sense to do a union for each event separately (i.e. 4688 and 4663) before the join. It is a more complete solution, though it would be rate that those two events are collected using different means.